labman/gui/templates/study_list.html

```diff
@@ -0,0 +1,69 @@
+{% extends sitebase.html %}
+{% block head %}
+<script type='text/javascript'>
+  var dtSelectedCounter = 0;
+
+  $(document).ready(function(){
+    var table = $('#studyListTable').DataTable(
+      {'columnDefs': [{'targets': 0, 'orderable': false, 'width': '30px'}],
+       'order': [[1, "desc"]],
+       'language': {'zeroRecords': 'No studies found'}});
+    $.get('/study_list', function(data) {
+      var datatable = $('#studyListTable').DataTable();
+      var newData = [];
+      for (var row of data.data) {
+        // Add the checkbox
+        var chBox = '<input type="checkbox" class="table-checkbox" data-lb-study-id="' + row[0] + '"></input>';
```

Comment:
@qiyunzhu Can you change this checkbox by the 'View study' button? Since we can only see 1 study at a time it doesn't make sense to allow the user to check multiple studies. This is the line that you've to modify, as we just discussed :-)

Reply:
@josenavas view button added. :)